vnext/Desktop/React.Windows.Desktop.vcxproj.filters

```diff
@@ -93,6 +93,12 @@
     <ClCompile Include="$(GeneratedFilesDir)module.g.cpp">
       <Filter>Generated Files</Filter>
     </ClCompile>
+    <ClCompile Include="WinHTTPWebSocketResource.cpp">
```

Comment:
>WinHTTPWebSocketResource [](start = 24, length = 24)

(nit) not a big deal, but since this is a new file, can we follow capitalization rules, so this should be WinHttpWebSocketResource.cpp

Rules are at:
[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)